app/classes/Transvision/ShowResults.php

```diff
@@ -315,20 +330,42 @@ public static function resultsTable($search_object, $search_results, $page)
                 $target_string2 = '';
             }
 
+            $string_id = md5($target_string);
+            $regular_string_id = 'string_' . $string_id;
+
+            // Find if we need to transliterate the string
+            $transliterate = ($locale2 == 'sr' && ! $extra_locale && $target_string)
```

Comment:
You can use `$transliterate = $locale2 == 'sr' && ! $extra_locale && $target_string != '@@missing@@';` no need to specify true when true and false when false ;)